src/py/mat3ra/made/tools/build/__init__.py

```diff
@@ -65,7 +65,14 @@ def _select(
     def _post_process(
         self, items: List[_GeneratedItemType], post_process_parameters: Optional[_PostProcessParametersType]
     ) -> List[Material]:
-        return [Material(self._convert_generated_item(item)) for item in items]
+        return [
+            (
+                self._convert_generated_item(item)
```

Comment:
what's happened here?

Reply:
If `_generate()` returns Material, then when the `_convert()` called we get `Material(Material)` which should be avoided.

Reply:
We should use `_GeneratedItemType == Material`